src/Controller/AssignController.php

```diff
@@ -94,7 +102,10 @@ public function assignForm(
             return $this->redirectToRoute('forms');
         }
 
-        $res = $assignService->assignForm($request, $form);
+        $success = $assignService->assignForm($request, $form);
+        if (!$success) {
+            $this->addFlash('danger', 'assignError');
```

Comment:
fixed, also I made the "required" tooltip on the summernote wysiwig editor visible, refactored the error translations and added e-mail sending if the assigneduser on a task changes via the form rather than the button.